api/v1beta1/placementapi_webhook.go

```diff
@@ -97,3 +118,30 @@ func (r *PlacementAPI) ValidateDelete() error {
 	// TODO(user): fill in your validation logic upon object deletion.
 	return nil
 }
+
+func (r PlacementAPISpec) ValidateCreate(basePath *field.Path) field.ErrorList {
+	return r.ValidateDefaultConfigOverwrite(basePath)
+}
+
+func (r PlacementAPISpec) ValidateUpdate(old PlacementAPISpec, basePath *field.Path) field.ErrorList {
+	return r.ValidateDefaultConfigOverwrite(basePath)
+}
```

Comment:
openstack-operator needs to call this

Reply:
Proposed in https://github.com/openstack-k8s-operators/openstack-operator/pull/660